src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      state.books = state.books.push({
```

Comment:
- [ OPTIONAL ] The following message is only to help you for better-written code, it does not mean that your code was bad, it was great :clap:  => :
The addBook reducer in the bookslice modifies the array of state.books by using the push method. However, as we know the push method returns the new length of the array and not the modified array itself so I would suggest that you set the state.boooks to the result of the state.books.push itself by using it directly:
`addBook: (state, { payload }) => {
  state.books.push({
    name: payload.name,
    author: payload.author,
    id: payload.id,
  });
},`

Reply:
@GzimAsani  Thank you for your recommendation, I implemented that method and it also fixed a linters issue.